compensated.cabal

```diff
@@ -87,3 +87,13 @@ test-suite doctests
       generic-deriving,
       semigroups     >= 0.9,
       simple-reflect >= 0.3.1
+
+benchmark criterion
+  type:           exitcode-stdio-1.0
+  main-is:        benchmark.hs
+  ghc-options:    -Wall
+  hs-source-dirs: tests
```

Comment:
Instead of putting this under `tests`, put this under a new `benchmarks` folder.